rcgen/src/key_pair.rs

```diff
@@ -1,13 +1,15 @@
 #[cfg(feature = "pem")]
 use pem::Pem;
-use ring::rand::SystemRandom;
-use ring::signature::KeyPair as RingKeyPair;
-use ring::signature::{self, EcdsaKeyPair, Ed25519KeyPair, RsaEncoding, RsaKeyPair};
 use std::convert::TryFrom;
 use std::fmt;
 use yasna::DERWriter;
 
 use crate::error::ExternalError;
+use crate::ring_like::error as ring_error;
+use crate::ring_like::rand::SystemRandom;
+use crate::ring_like::signature::KeyPair as RingKeyPair;
+use crate::ring_like::signature::{self, EcdsaKeyPair, Ed25519KeyPair, RsaEncoding, RsaKeyPair};
```

Comment:
WDYT about folding the L10 import into L11's grouping since they share namespaces.
```suggestion
use crate::ring_like::signature::{
	self, EcdsaKeyPair, Ed25519KeyPair, KeyPair as RingKeyPair, RsaEncoding, RsaKeyPair,
};
```